common/test/utils.py

```diff
@@ -0,0 +1,55 @@
+"""Utilities for testing"""
+from pathlib import Path
+from collections import namedtuple
+
+SnapshotConfig = namedtuple('SnapshotConfig', [
+    'config_version',
+    'snapshot_type',
+    'snapshot_value',
+    'snapshot_size',
+    'no_on_battery',
+    'notify_enabled',
+    'snapshot_path',
+    'snapshot_host',
+    'snapshot_profile',
+    'snapshot_user',
+    'preserve_acl',
+    'preserve_xattr',
+    'remove_old_snapshots_enabled',
+    'remove_old_snapshots_unit',
+    'remove_old_snapshots_value',
+    'rsync_options_enabled',
+    'rsync_options_value',
+    'profiles_version'
+])
+
+
+def generate_temp_config(snapshot_config: SnapshotConfig) -> str:
```

Comment:
I don't understand the value of that solution. Can you explain?
And beside that you use it only twice in the code base.